silkrpc/ethdb/kv/remote_transaction.hpp

```diff
@@ -69,9 +70,6 @@ class RemoteTransaction : public Transaction {
     }
 
     asio::awaitable<void> close() override {
-        for (const auto& [table, cursor] : cursors_) {
-            co_await cursor->close_cursor();
-        }
```

Comment:
I would prefer this change to come from PR #191 because we should try to keep each PR as much scoped as possible